config/runtime/tests/fault-injection.jinja2

```diff
@@ -0,0 +1,32 @@
+- test:
+    timeout:
+      minutes: {{ job_timeout|default(20) }}
+    definitions:
+    - from: inline
+      name: "fault-injection-{{ test_name }}"
+      path: inline/fault-injection.yaml
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: fault-injection
+          description: "Fault injection test: {{ test_name }}"
+          os:
+          - debian
+          scope:
+          - functional
+        run:
+          steps:
+            - wget {{ failcmd_url }} -O /tmp/failcmd.sh
+            - chmod +x /tmp/failcmd.sh
```

Comment:
Maybe we can wrap the wget and chmod commands in a LAVA test set, so it's easier to spot a failure if it happens in this case. So something like:
```
- lava-test-set start setup
- <wget command>
- <chmod command>
- lava-test-set stop setup
```